addons/io_hubs_addon/debugger.py

```diff
@@ -153,8 +153,8 @@ def execute(self, context):
 
 class HubsOpenRoomOperator(bpy.types.Operator):
     bl_idname = "hubs_scene.open_room"
-    bl_label = "Open Room"
-    bl_description = "Open room"
+    bl_label = "Open selected room"
+    bl_description = "Opens the selected room"
```

Comment:
Maybe some additional info, like this:
```suggestion
    bl_description = "Opens the selected room in the browser selected in the add-on preferences.  The specified room flags will be applied"
```

Reply:
Updated